src/server/controller/user.js

```diff
@@ -39,7 +39,7 @@ router.get('/auth/github', checkReturnTo);
 // router.get('/auth/github/callback', passport.authenticate('github', { successReturnToOrRedirect: '/', failureRedirect: '/' }));
 router.get('/auth/github/callback', passport.authenticate('github', { failureRedirect: '/' }),
     function (req, res) {
-        if(req.user && req.session.requiredScope != 'public' && ( !req.user.scope || req.user.scope.indexOf('write:repo_hook') < 0 )) {
+        if(req.user && req.session.requiredScope != 'public' && req.user.login == config.server.github.admin_user && ( !req.user.scope || req.user.scope.indexOf('write:repo_hook') < 0 )) {
```

Comment:
This code does the trick, but I'm wondering how I can go about making it read better —

```javascript
  if(req.user && req.session.requiredScope != 'public' && ( !req.user.scope || req.user.scope.indexOf('write:repo_hook') < 0 ) && 
    (config.server.github.admin_users.length > 0 && config.server.github.admin_users.indexOf(req.user.login) >= 0)) {
    return res.redirect('/auth/github?admin=true');
  }
```